pkg/kn/commands/service/describe.go

```diff
@@ -171,6 +171,9 @@ func writeService(dw printers.PrefixWriter, service *v1alpha1.Service) {
 	if (service.Spec.Template != nil) && (service.Spec.Template.Spec.ServiceAccountName != "") {
 		dw.WriteAttribute("ServiceAccount", service.Spec.Template.Spec.ServiceAccountName)
 	}
+	if service.Spec.Template != nil && service.Spec.Template.Spec.ImagePullSecrets != nil {
+		dw.WriteAttribute("ImagePullSecrets", service.Spec.Template.Spec.ImagePullSecrets[0].Name)
```

Comment:
I wonder whether we should use the label with spaces (like in "Image Pull Secrets", maybe also for "Service Account") as we use this for other parts in describe, too, for an easier consumption by a human reader.

But I just saw, that even for a `kubectl describe` this is not consistent (i.e. mixed usage of camel-case and space separated labels). So probably not a big thing :)

Reply:
It occurred to me as well, though I went along with other fields as mentioned for ServiceAccount. Will change it to space separated.

Reply:
Changed to `Image Pull Secrets` and `Service Account`.